panther_analysis_tool/main.py

```diff
@@ -311,10 +311,16 @@ def test_analysis(args: argparse.Namespace) -> Tuple[int, list]:
         list(load_analysis_specs(args.path)) +
         list(load_analysis_specs(HELPERS_LOCATION)))
 
+    if len(analysis) == 0:
+        return 1, "No valid analyses found in {}".format(args.path)
```

Comment:
```suggestion
        return 1, "No tests found in {}, skipping!".format(args.path)
```

Reply:
what do you think about `tests` vs `analyses`?

Reply:
I changed to something similar to this.